db_builder/Source/main.cpp

```diff
@@ -38,15 +55,33 @@ int main()
     sqlite3_exec(db.get(), "BEGIN TRANSACTION;", NULL, NULL, NULL);
     createTables(db);
 
+    fd::FileDownloader downloader;  
+    downloader.addDirectoryPath(directoryPath_);
+    
+    // Downloading Zips with images
+    // adding info for downloading (links, file names)
+    std::string links_[] = {"https://dd.b.pvp.net/latest/set1-lite-en_us.zip"
+                           ,"https://dd.b.pvp.net/latest/set2-lite-en_us.zip"
+                           ,"https://dd.b.pvp.net/latest/set3-lite-en_us.zip"
+                           ,"https://dd.b.pvp.net/latest/set4-lite-en_us.zip"};
+    std::string fileNames_[] = {"set1-lite-en_us.zip"
+                               ,"set2-lite-en_us.zip"
+                               ,"set3-lite-en_us.zip"
+                               ,"set4-lite-en_us.zip"};
+    downloader.addLinks(links_, fileNames_, (sizeof(links_)/sizeof(links_[0])));
+    
+    downloader.download(true);
+    
+    
     // temporary ugly shit. We are going to abadon it for sake of auto downloading
     Json::Value globalsJson =
-        getJsonFromFile("globals-ru_ru.json", "https://dd.b.pvp.net/latest/core/ru_ru/data/globals-ru_ru.json");
+        getJsonFromFile("globals-en_us.json", "https://dd.b.pvp.net/latest/core/en_us/data/globals-en_us.json");
```

Comment:
tutaj na pewno zamianimy na en_us ? Pytam, bo kojarze że ktoś mówił, że ten rosysjki tutaj był z jakiegoś powodu

Reply:
rosyjski z tego co kojażę to był tylko do testów